src/FlaUI.Core/Application.cs

```diff
@@ -13,7 +13,7 @@ namespace FlaUI.Core
 {
     public class Application : IDisposable
     {
-        private static readonly ILogger Log = new ConsoleLogger();
+        public ILogger Log { get; }

```

Comment:
Yeah, I think it is a good idea. It allows one to just over ride the logger for a single application.